tests/test_modeling.py

```diff
@@ -335,6 +337,22 @@ def test_compare_to_transformers_non_dyn_bs(self, model_arch):
 
         gc.collect()
 
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    def test_pipeline_model(self, model_arch):
+        model_args = {"test_name": model_arch, "model_arch": model_arch}
+        self._setup(model_args)
+
+        model_id = self.ARCH_MODEL_MAP[model_arch] if model_arch in self.ARCH_MODEL_MAP else MODEL_NAMES[model_arch]
```

Comment:
Are we testing on "tiny" models to make this not too slow?

Reply:
we are testing the same models as for normal inference. just to make sure the pipeline works. Models were added by @JingyaHuang. It seems to be tiny models if not explicitly defined.